Documentation/EyeTracking/EyeTracking_TargetSelection.md

```diff
@@ -12,7 +12,7 @@ This page discusses different options for accessing eye gaze data and eye gaze s
 
 To select holographic content using eye gaze, there are several options:
 
-**1. Use the primary focus pointer:**
+[**1. Use the primary focus pointer:**](EyeTracking_TargetSelection.md#1-use-generic-focus-and-pointer-handlers)
```

Comment:
>EyeTracking_TargetSelection.md [](start = 40, length = 30)

I think this part of the thing can be omitted since it's in the same file

Reply:
Updated! :)